### DISCUSSION TITLE
Attachements to Records

### DISCUSSION THREAD

**Discussion by RyanMountain** [netbox-community/netbox#10674] [Ideas]
Hi Everyone,

I think it would be a nice feature if we were able to add attachments to certain tables/areas within NetBox. For example, in Sites, Tenants, Circuits to name a few.

Reason behind this is you may have to sign an agreement to lease the building so want to store a copy of this document so where else better to put it than under the site in NetBox.

Similarly goes for circuits when you sign the order or when you have an agreement with a tenant etc.

**Comment by jcollie** [comment]
Try the netbox-documents plugin. https://github.com/jasonyates/netbox-documents
